spec/lib/ryan.rb

```diff
@@ -0,0 +1,46 @@
+require "spec_helper"
+# rspec spec/lib/ryan.rb
+describe Ryan do
+  klass = Ryan
```

Comment:
haha I guess you could do `let(:klass) { described_class }`. I'm fine though with leaving it as-is